src/main/java/com/timgroup/statsd/NonBlockingStatsDClientBuilder.java

```diff
@@ -129,17 +147,25 @@ public NonBlockingStatsDClientBuilder telemetryFlushInterval(int val) {
      * @return the built NonBlockingStatsDClient.
      */
     public NonBlockingStatsDClient build() throws StatsDClientException {
-        if (addressLookup != null) {
-            return new NonBlockingStatsDClient(prefix, queueSize, constantTags, errorHandler,
-                    addressLookup, timeout, socketBufferSize, maxPacketSizeBytes, entityID,
-                    bufferPoolSize, processorWorkers, senderWorkers, blocking, enableTelemetry,
-                    telemetryFlushInterval);
-        } else {
-            return new NonBlockingStatsDClient(prefix, queueSize, constantTags, errorHandler,
-                    staticStatsDAddressResolution(hostname, port), timeout, socketBufferSize, maxPacketSizeBytes,
-                    entityID, bufferPoolSize, processorWorkers, senderWorkers, blocking, enableTelemetry,
-                    telemetryFlushInterval);
+        Callable<SocketAddress> lookup = addressLookup;
+        Callable<SocketAddress> telemetryLookup = null;
```

Comment:
`Callable<SocketAddress> telemetryLookup = telemetryAddressLookup`?

Reply:
Yes! Not sure what happened! 🤦